types/stylelint/index.d.ts

```diff
@@ -98,6 +98,7 @@ declare namespace stylelint {
 		ignorePatterns?: string;
 		rules?: ConfigRules;
 		quiet?: boolean;
+		quietDeprecationWarnings?: boolean;
```

Comment:
[suggest] I'm not sure why `quiet` exists in the configuration, but `quietDeprecationWarnings` does not need to exist in the config.

By removing `quietDeprecationWarnings` from the config, the code would be much simpler.

```suggestion
```